documentation/content/en/books/handbook/cutting-edge/_index.adoc

```diff
@@ -245,11 +245,34 @@ Rebuilding a custom kernel, even if nothing else changed, allows `uname` to accu
 This is particularly helpful when maintaining multiple systems, as it allows for a quick assessment of the updates installed in each one.
 
 [[freebsdupdate-upgrade]]
-=== Performing Major and Minor Version Upgrades
+=== Performing Minor and Major Version Upgrades
 
-Upgrades from one minor version of FreeBSD to another, like from FreeBSD 9.0 to FreeBSD 9.1, are called _minor version_ upgrades.
-_Major version_ upgrades occur when FreeBSD is upgraded from one major version to another, like from FreeBSD 9.X to FreeBSD 10.X.
-Both types of upgrades can be performed by providing `freebsd-update` with a release version target.
+Upgrades from one minor version of FreeBSD to another are called _minor version_ upgrades. An example: 
+
+- FreeBSD 13.1 to 13.2.
+
+_Major version_ upgrades increase the major version number. An example: 
+
+- FreeBSD 12.4 to 13.2.
+
+Both types of upgrade can be performed by providing `freebsd-update` with a release version target.
+
+[WARNING]
+====
+After each new `RELEASE`, FreeBSD package build servers will, for a limited period, *not* use newest version of the operating system.
```

Comment:
For future reference: 

* this first sentence is not accurate – not /entirely/ true. 

If I had made the paragraph accurate, the (relative) complexity would have detracted from the key point of the warning. 

----

Generally, I try to avoid warning boxes and note boxes – their presence sometimes signals an underlying need to restructure things. 

In this case, I chose a warning box because we were approaching the date of release of 13.2. 

(It's easier/quicker to gain approval to commit a condensed warning, than it is to gain approval for a broader restructure.)